src/commands/admin/issue.ts

```diff
@@ -1,83 +1,65 @@
 /* eslint-disable camelcase */
-import { adminPerms } from '@lib/permissions';
+import { ADMIN_PERMS } from '@lib/permissions';
 import { RequestError } from '@octokit/types';
 import { BOT, GITHUB_PROJECT } from '@root/config';
 import { Command } from '@lib/types/Command';
-import { Message } from 'discord.js';
+import { ApplicationCommandOptionData, ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
 
 export default class extends Command {
 
-	usage = '<title>|[options]';
-	description = `Creates an issue in ${BOT.NAME}' repository.`;
-	extendedHelp = `You must pass in an issue title. Flags can be any of:
-	--labels=[comma, separated, list]\n--milestone=[milestone number]\n--project=[project (defaults to SageV2)]]`;
+	description = `Creates an issue in ${BOT.NAME}'s repository.`;
+	tempPermissions: ApplicationCommandPermissionData[] = [ADMIN_PERMS];
 
-	async permissions(msg: Message): Promise<boolean> {
-		return adminPerms(msg);
-	}
+	options: ApplicationCommandOptionData[] = [{
+		name: 'title',
+		description: 'What\'s the issue?',
+		type: 'STRING',
+		required: true
+	},
+	{
+		name: 'labels',
+		description: 'The issue labels',
+		type: 'STRING',
+		required: false
+	},
+	{
+		name: 'body',
+		description: 'The issue body',
+		type: 'STRING',
+		required: false
+	}]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		const title = interaction.options.getString('title');
+		const label = interaction.options.getString('labels');
+		const body = interaction.options.getString('body');
 
-	async run(msg: Message, [title, project, labels, milestone, body]: [string, string, string[], string, string]): Promise<Message | void> {
-		const newIssue = await msg.client.octokit.issues.create({
+		const labels = label ? label.split(', ') : [];
+
+		const newIssue = await interaction.client.octokit.issues.create({
 			owner: 'ud-cis-discord',
-			repo: project,
+			repo: GITHUB_PROJECT,
 			title: title,
-			milestone: milestone || undefined,
 			labels: labels,
-			body: `${body}\n\n<sub>Created by ${msg.member.displayName} via ${BOT.NAME}</sub>`
+			body: body || `\n\n<sub>Created by ${interaction.user.username} via ${BOT.NAME}</sub>`
 		}).catch(response => {
 			console.log(response);
 			let errormsg = '';
 			const { errors } = response as RequestError;
 			errors.forEach(error => {
 				errormsg += `Value ${error.code} for field ${error.field}.\n`;
 			});
-			msg.channel.send(`Issue creation failed. (HTTP Error ${response.status})
+			interaction.channel.send(`Issue creation failed. (HTTP Error ${response.status})
```

Comment:
This can probably just be a `return interaction.reply(...)` instead of sending a separate message